binding.cc

```diff
@@ -90,6 +92,28 @@ void ExitOnPromiseRejectCallback(PromiseRejectMessage promise_reject_message) {
   exit(1);
 }
 
+MaybeLocal<Module> ResolveCallback(Local<Context> context,
+                                   Local<String> name,
+                                   Local<Module> referrer) {
+  auto isolate = Isolate::GetCurrent();
+  worker* w = (worker*)isolate->GetData(0);
```

Comment:
I think you should add 
`HandleScope handle_scope(isolate);`
here?

Reply:
I don't actually understand what handle_scope does. Haha.